package.json

```diff
@@ -18,46 +18,26 @@
   "author": "Aliaksei Hrynko (https://github.com/hrynko)",
   "repository": "github:hrynko/vue-pdf-embed",
   "scripts": {
-    "prepare": "webpack build",
-    "build": "webpack build --progress",
-    "serve": "webpack serve --config webpack.dev.config.js --hot",
-    "test": "jest",
+    "build": "vite build -c vite.library.config.js",
+    "demo": "vite",
     "lint": "eslint \"**/*.{js,vue}\"",
     "lint:fix": "eslint \"**/*.{js,vue}\" --fix"
   },
+  "dependencies": {
+    "pdfjs-dist": "^2.9.359"
+  },
```

Comment:
Is there any special meaning in moving this dependency here? Can we make the version more flexible and move it to `peerDependencies`?

Reply:
I think it's just a case of clarity, showing that it has a clear dependecy on that lib and version. It's not a dev dependency, even tho the end result is bundled up in the package.

Not sure if peer is more useful as you likely don't want the user to be concerned about this in 99% of cases.

Reply:
I definitely don't want the user to be concerned about this when using UMD, but in the case of ESM it would be great to provide the ability to select the PDF.js version manually. There was a request for this in the issues. Do you think this is reasonable?

Reply:
It depends, as switching the PDFjs version could change the interface, so would (likely) require other changes to this lib by the user? It might be good to have another PR after this one to move to the latest pdf-js though. If this lib supported 3.x, in theory the dependency could be any pdfjs version that starts 3., and it could then be changed by the user

Reply:
It would be nice, but ok, let's keep it for another PR